api/models.py

```diff
@@ -2,18 +2,20 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
 from django.contrib.postgres.fields import JSONField
+from typedmodels.models import TypedModel
 
 
 class User(AbstractUser):
     id = models.AutoField(primary_key=True)
     gravatar = models.URLField(blank=True)
 
 
-class Collection(models.Model):
+class CollectionBase(TypedModel):
     title = models.TextField()
     description = models.TextField(null=True, blank=True)
     tags = models.TextField(null=True, blank=True)
     created_by = models.ForeignKey(User)
+    created_by_org = models.TextField(null=True, blank=True)
```

Comment:
Update from conversation with Casey and Chris: TextField and CharField are treated the [same](https://www.postgresql.org/docs/9.0/static/datatype-character.html) by Postgres. That being said, I think best practice, should we ever move to a different backend for whatever reason, would be to stick with CharField for the lil guys and TextField for the big guys